src/landscape/components/LandscapeForm/BoundaryStep.js

```diff
@@ -160,7 +160,10 @@ const BoundaryOptions = props => {
             key={index}
             fullWidth
             variant="outlined"
-            onClick={option.onClick}
+            onClick={() => {
```

Comment:
Suggestion: Move the on click handler outside of the markup, I think it is more readable if you have more than one action performing here to move it outside